### ISSUE TITLE
Log note for configurations with oauth and encrypted values

### ISSUE THREAD

**Issue by ErikZigo** [keboola/app-project-restore#6]

**Comment by Halama** [comment]
Mě Oauth po migraci funguje korekntě, UI nenajde záznam pod daným id tak nabizí autorizaci.
![image](https://user-images.githubusercontent.com/903531/40475248-68407606-5f41-11e8-8dc1-8aefd6c9d4fe.png)

Tobě se to někde chovalo jinak?

**Comment by Halama** [comment]
U obyčejné zašifrované hodnoty to zařve takhle, což je taky ok
![image](https://user-images.githubusercontent.com/903531/40475348-aaf91e58-5f41-11e8-91b8-c6e79c637320.png)

**Comment by Halama** [comment]
ještě jsem se bavil s @kacurez a asi by bylo lepší to u toho oauth odmazat z té konfigurace

**Comment by ErikZigo** [comment]
Ja mam dojem ze mi to delalo kdyz jsem migroval ve stejnem regionu. Tzn oauth s tim id uz existoval v jinem projektu. Poustim si ted schvalne migraci ve stejnem regionu.

**Comment by Halama** [comment]
jo jasny, vymazem to, bude to cistsi

**Comment by ErikZigo** [comment]
Tak kecam. Kdyz migruju ve stejnem regionu a vlezu do konfigurace, tak tam taky vidim ze mam udelat novou OAuth autorizaci

**Comment by Halama** [comment]
Fixnu to. Jenom teda to že dá pryč ten `authorization.oauth_api`

**Comment by ErikZigo** [comment]
ok

**Comment by ErikZigo** [comment]
Koukam ze to delas primo do te php libky.
Ja myslim ze by to tam byt nemelo a melo by to byt soucasti tyhle aplikace. 

Zamyslel jsem to tak ze to vylistuje vsechny restornute konfigurace a ty pripadne upravi.

**Comment by Halama** [comment]
aha, me to docela davalo smysl tam protoze pri tom restore to bude potreba vzdy udelat takhle.

https://github.com/keboola/php-kbc-project-restore/pull/12

**Comment by ErikZigo** [comment]
Tu libku pak ale bude vyuzivat i sapi CLI, kde se tohle dit nebude. Ale klidne to tam nech :-) Pripadne pri refactoringu co mam v planu to casem kdyztak prenesu.

**Comment by Halama** [comment]
jo, ale u SAPI cli si myslim že to bude stejná situace. Někam restorneš projekt, tak tam stejně ten oauth nebude. A pokud by se na to našel use case tak se to kdyžtak zas přesune.
